libevm/VMOpt.cpp

```diff
@@ -119,9 +126,10 @@ void VM::optimize()
 		}
 #endif
 	}
-	
-	copyCode(pc - nBytes + 33);
```

Comment:
Ok, so now we won't need this `pc = nBytes` since we don't use `pc` after the loop.
And I moved `pc` declaration into the for loop, because it won't be needed outside.